### ISSUE TITLE
Deleting files moves them to Trash

### ISSUE THREAD

**Issue by Ap0them** [lutris/lutris#4314]
I just deleted a game stored on ~/Windows/*game name* and upon deleting it, it also deleted my ~/Windows folder, which holds all my games and their saves. So because of this I lost a few hundred gbs of games and saves. This could be avoided by simply moving files to trash, thank you.

**Comment by belkka** [comment]
Just lost my ~/Lutris/wine64 wine prefix because aborted new game installation

**Comment by vince6794** [comment]
Same here. 
Just lost a bunch of GBs of games and most importantly... hundred of hours of savegame :-1:  

Any ETA on this ?

**Comment by danieljohnson2** [comment]
This is not a simple thing to do. Python does not support it, and it is much more complicated than just moving a file.

[The spec is here](https://specifications.freedesktop.org/trash-spec/trashspec-latest.html) if you are interested.

Lutris warns when you are deleting things it thinks you shouldn't, but this works a lot better if you use the default game directories (~/Games).

**Comment by strycore** [comment]
Lutris no longer deletes files from cancelled installation that were present before the install started.
It also no longer deletes files of games manually installed.

We have plans to generate file lists during installs and keep them around for the uninstall process when we will only delete the files present in that list.

Hijacking this issue with this since we don't really plan to use the trash at any point.

**Comment by legluondunet** [comment]
@strycore this feature request can be closed? Latest Lutris version already move games to trash when they are deleted from Lutris interface right?

**Comment by strycore** [comment]
no, I still want to implement this at some point, shouldn't be too hard
